build/ci/job-template.yml

```diff
@@ -181,36 +181,6 @@ jobs:
         pathToPublish: $(Build.ArtifactStagingDirectory)
         artifactName: ${{ parameters.name }} $(_config_short)
         artifactType: container
-    - ${{ if and(eq(parameters.nightlyBuild, 'false'), eq(parameters.buildScript, 'build.cmd')) }}:
-      - powershell: |
```

Comment:
Wasn't this added to conserve disk space? Why is it not needed anymore?

Reply:
Yes, it is added to save disk space only for windows platform long time ago and I have seen this step failing some times.
Since we have optimized disk usage like use symbol links and optimized package disk usage it is not necessary any more.

---
In reply to: [448137620](https://github.com/dotnet/machinelearning/pull/5272#discussion_r448137620) [](ancestors = 448137620)